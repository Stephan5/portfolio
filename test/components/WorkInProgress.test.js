import React from "react";
import ReactDOM from "react-dom";
import WIP from "components/MenuBar/MenuBar";
import renderer from "react-test-renderer";

test("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<WIP/>, div);
  ReactDOM.unmountComponentAtNode(div);
});

test("snapshot", () => {
  const component = renderer.create(
    <WIP/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
