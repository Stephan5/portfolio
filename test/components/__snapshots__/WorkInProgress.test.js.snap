// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="ui inverted ten item menu"
>
  <div
    className="header item"
    onClick={[Function]}
  >
    Stephan's World
  </div>
  <div
    className="item"
    onClick={[Function]}
  >
    Starling
  </div>
  <div
    className="disabled item"
    onClick={[Function]}
  >
    Portfolio
  </div>
  <div
    className="right menu"
  />
</div>
`;
